src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,14 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+      {' by '}
```

Comment:
you can wrap with `span`
```suggestion
      <span> by </span>
```